integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -1014,4 +1025,40 @@ public void tearDownAll() {
     }
   }
 
+  /**
+  *  JUnit5 extension class to implement ExecutionCondition for the custom
+  *  annotation @AssumeWebLogicImage.
+  */
+  private static class WebLogicImageCondition implements ExecutionCondition {
+
+    /**
+     * Determine if the the test "testUpdateImageName" will be skipped based on WebLogic image tag.
+     * Skip the test if the image tag is the latest one.
+     *
+     * @param context the current extension context
+     * @return ConditionEvaluationResult disabled if the image tag is the latest one, enabled if the
+     *         image tag is not the latest one
+    */
+    @Override
+    public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext context) {
+      if (WEBLOGIC_IMAGE_TAG.equals(WLS_LATEST_IMAGE_TAG)) {
+        getLogger().info("WebLogic image tag is {0}. No latest image available to continue test. Skipping test",
+            WLS_LATEST_IMAGE_TAG);
+        return ConditionEvaluationResult
+            .disabled(String.format("No latest image available to continue test. Skipping test!"));
+      } else {
+        getLogger().info("Updating image to {0}. Continuing test!", WLS_UPDATE_IMAGE_TAG);
+        return ConditionEvaluationResult
+            .enabled(String.format("Updating image to {0}. Continuing test!", WLS_UPDATE_IMAGE_TAG));
+      }
+    }
+  }
+
+  @Target({ElementType.TYPE, ElementType.METHOD})
+  @Retention(RetentionPolicy.RUNTIME)
+  @Tag("assume-weblogic-image")
```

Comment:
Do we need this annotation? There is no annotation called "assume-weblogic-image" right?

Reply:
Right now there is no annotation called "assume-weblogic-image". But i think it doesn't hurt to add Tag name in case we need image control later.

Reply:
Right now we don't use "assume-weblogic-image" but I think it won't hurt to add Tag name in case later we need another image control or test cases grouping, just like @Tag("must-not-run-in-parallel") or @Tag("integration")

Reply:
ok alright.